website/docs/r/instance_ip_reverse_dns.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_instance_ip_reverse_dns"
+description: |-
+  Manages Scaleway Compute Instance IPs reverse DNS.
+---
+
+# scaleway_instance_ip_reverse_dns
+
+Manages Scaleway Compute Instance IPs Reverse DNS.
+
+## Example Usage
+
+```hcl
+resource "scaleway_instance_ip" "server_ip" {}
+resource "scaleway_instance_ip_reverse_dns" "reverse" {
+  ip_id = scaleway_instance_ip.server_ip.id
+  reverse = "www.scaleway.com"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `ip_id` - (Required) The IP ID
+- `reverse` - (Required) The reverse DNS for this IP.
+- `zone` - (Defaults to [provider](../index.html#zone) `zone`) The [zone](../guides/regions_and_zones.html#zones) in which the IP should be reserved.
+
+## Attributes Reference
+
+In addition to all above arguments, the following attributes are exported:
+
+- `id` - The ID of the IP.
```

Comment:
Reverse as well

Reply:
reverse is part of `all above arguments`